_docs/0.3/usage.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+title: Usage
+version: '0.3'
+categories:
+    - '0.3'
+redirect_from: /docs/usage
+---
+#### Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Dataset Comparison](#dataset-comparison)
+- [Info File Comparison](#info-file-comparison)
+- [E2E Runner](#e2e-runner)
+- [Plugins](#plugins)
+
+***
+
+#### Dataset Comparison
+
+Dataset comparison can be run as a spark job or used as a library.
+
+Dataset Comparison used as a spark job example. _This example doesn't show spark argumetns_
+Example:
+
+```shell
+spark-submit dataset-comparison.jar \
+    --ref-format csv \
+    --ref-path /path/to/csv \
```

Comment:
This example does not make it clear if the folder or the file should be entered. Maybe be verbose as
```suggestion
    --ref-path /path/to/csv-dir \ // will pickup `data.csv` in the directory
```
or 
```suggestion
    --ref-path /path/to/data.csv \
```